ember-toucan-core/src/components/form/checkbox-group-field.ts

```diff
@@ -0,0 +1,77 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+import CheckboxFieldComponent from './checkbox-field';
+
+import type { WithBoundArgs } from '@glint/template';
+
+export interface ToucanFormCheckboxGroupFieldComponentSignature {
+  Element: HTMLFieldSetElement;
+  Args: {
+    /**
+     * Provide a string to this argument to render an error message and apply error styling to the Field.
+     */
+    error?: string;
+
+    /**
+     * Provide a string to this argument to render a hint message to help describe the control.
+     */
+    hint?: string;
+
+    /**
+     * Sets the disabled attribute on the fieldset.
+     */
+    isDisabled?: boolean;
+
+    /**
+     * Provide a string to this argument to render inside of the label tag.
+     */
+    label: string;
+
+    /**
+     * Sets the name attribute of the checkboxes. A string specifying a name for the input control. This name is submitted along with the control's value when the form data is submitted.
+     *
+     * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name
+     */
+    name: string;
+
+    /**
+     * The function called when a checkbox element is clicked.
+     */
+    onChange?: (selectedValues: Array<string>, e: Event | InputEvent) => void;
+
+    /**
+     * The currently selected checkbox elements. The elements with the matching `@option` / underlying value attribute will be checked.
+     */
+    value?: Array<string>;
+  };
+  Blocks: {
+    default: [
+      {
+        CheckboxField: WithBoundArgs<
+          typeof CheckboxFieldComponent,
+          'isDisabled' | 'name' | 'value'
```

Comment:
Is this correct? According to the template, it should be [these four arguments](https://github.com/CrowdStrike/ember-toucan-core/pull/81/files#diff-f513e3b13a120c52fd2a93f32c4f56af79da3ded446cbb4ebae0a1c2b9a0717aR13-R16)?

Wonder why Glint seems happy though!? 🤔

Reply:
Oh my word, excellent eyes!  I forgot to update these when iterating.  That's super interesting.  I do wonder why Glint didn't complain.  Think this is Glint issue worthy?

Reply:
Sounds excellent to me! I'm going to go ahead and merge this as is if that's okay with you, but we should chat about your findings on this and maybe https://github.com/CrowdStrike/ember-toucan-core/pull/81#discussion_r1143491402 as follow ups later this week if you have some time! 🙇